sonar-db/src/test/java/org/sonar/db/component/ResourceIndexDaoTest.java

```diff
@@ -148,4 +149,23 @@ public void select_project_ids_from_query_and_view_or_sub_view_uuid() {
     assertThat(underTest.selectProjectIdsFromQueryAndViewOrSubViewUuid(session, "two", viewUuid)).containsOnly(2L);
     assertThat(underTest.selectProjectIdsFromQueryAndViewOrSubViewUuid(session, "unknown", viewUuid)).isEmpty();
   }
+
+  /**
+   * SONAR-7594
```

Comment:
- you think the reference to the jira ticket is important?
- same question for the description